README.md

```
@@ -1,338 +1,253 @@
-  Display autocomplete installation instructions.
-
-EXAMPLES
-  $ mplx autocomplete
+## Table of Contents
 
-  $ mplx autocomplete bash
+- [Installation](#installation)
+- [Quick Start](#quick-start)
+- [Command Structure](#command-structure)
+- [Documentation](#documentation)
+- [Development](#development)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)